src/devices/branch.jl

```diff
@@ -0,0 +1,320 @@
+
+function get_device_to_pm(
+    ix::Int,
+    branch::PSY.PhaseShiftingTransformer,
+    device_formulation::Type{D},
```

Comment:
We should handle the formulation aspect in PSI and remove these fields from this package.